### PR TITLE
Add tariff info to customer profile

### PR DESCRIPTION
**Pull request by wangannie**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

- Added the tariff info section to the top of the customer profile
  - I just adjusted the `OutlinedCardList` to have a `columns` prop (not sure if this is a very intuitive name since technically the whole cardlist becomes a row...which is the opposite..anyways I'm open to suggestions) which basically switches it to lay the stats side by side.
  - If, for any reason, a tariff value is blank/invalid, it should show `-`
- Minor styling updates to bring it closer to the designs -- final front-end changes are on hold pending finalized designs.

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review

Run the `annie/tariff-plans` in the node app ([Node PR #6](https://github.com/calblueprint/meepanyar-node/pull/6)) to get the updated tariff info.

### Online sources

https://material-ui.com/styles/basics/#adapting-based-on-props

### Related PRs

https://github.com/calblueprint/meepanyar-node/pull/6

## Next steps
- Update/finalize styling!

### Screenshots

**Normal behavior**
image

**Edge case: missing/invalid values**
image

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'

### CODE REVIEW

File: src/components/OutlinedCardList.tsx [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+  label: string;
+  unit: string;
+}
+
+interface CardProps {
+  info: CardPropsInfo[];
+  primary: boolean;
+  rightIcon?: JSX.Element;
+  columns?: boolean;
+}
```
Comment (julianrkung) [inline src/components/OutlinedCardList.tsx:16]: Is there a reason this was moved to be above the styles? If not can we move it back to preserve blame history?

File: src/lib/airtable/interface.ts [hunk 2]
```diff
@@ -46,8 +47,8 @@ export interface TariffPlanRecord {
 export interface CustomerRecord {
   name: string;
   meterNumber: number;
-  tariffPlansId: string[];
-  tariffPlans: TariffPlanRecord[];
+  tariffPlansId: string;
+  tariffPlans: TariffPlanRecord;
```
Comment (julianrkung) [inline src/lib/airtable/interface.ts:51]: Yeah this naming is annoying and confusing I'll change it during worksession tonight, sorry!

**Review by julianrkung — APPROVED** [review]
LGTM! Thanks for getting this done, good work :)

I wasn't too stringent on the styling because designs aren't finalized, but if this design is what we end up with, we should standardize the tariff portion (IMO it's offsetting that sometimes the `MMK` units are on a 3rd line and sometimes on the 2nd line)

File: src/components/OutlinedCardList.tsx (hunk 1 above)
Comment (wangannie) [reply src/components/OutlinedCardList.tsx:16]: Hm I thought I had to move this because I use the props in `makeStyles`

Comment (julianrkung) [reply src/components/OutlinedCardList.tsx:16]: It still compiles and runs if i move it back (I think interfaces might be hoisted?). 

Actually, it's probably better practice to just put all types at the top (more important than preserving blame in this case). I'm fine with this diff, sorry!

**Comment by wangannie** [comment]
Good catch! Updated to move unit to a new line when it's in this column layout.
image

**Pull request merged**